src/components/FetchedPosts.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import Post from './Post'
+import { fetchHeros } from '../redux/actions'
+import { Loader } from './Loader'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+export default () => {
+	const dispatch = useDispatch()
+	const posts = useSelector(state => state.posts.fetchedHeros)
```

Comment:
create file someName.selectors.js and store selectors there. Use 'reselect' library
 Example:
```
 import {createSelector} from 'reselect';

export const appSelector = (state) => state.app;
export const mainNavStateSelector = createSelector(
	appSelector,
	(state) => state.get('navMenuExpanded'),
);
export const snackBarMessagesSelector = createSelector(
	appSelector,
	(state) => state.get('messages'),
);

export const isAppLoadingSelector = createSelector(
	appSelector,
	(state) => state.get('isLoading'),
);
```
After that you can use selectors as:
`const items = useSelector(mainNavStateSelector);`